spot_wrapper/spot_arm.py

```diff
@@ -0,0 +1,554 @@
+import time
+import typing
+import logging
+
+from bosdyn.util import seconds_to_duration
+from bosdyn.client.async_tasks import AsyncPeriodicQuery
+from bosdyn.client import robot_command
+from bosdyn.client.robot import Robot
+from bosdyn.client.robot_state import RobotStateClient
+from bosdyn.client.robot_command import RobotCommandBuilder, RobotCommandClient
+from bosdyn.client.image import ImageClient, build_image_request
+from bosdyn.client.manipulation_api_client import ManipulationApiClient
+from bosdyn.api import robot_command_pb2
+from bosdyn.api import arm_command_pb2
+from bosdyn.api import synchronized_command_pb2
+from bosdyn.api import geometry_pb2
+from bosdyn.api import trajectory_pb2
+from bosdyn.api import manipulation_api_pb2
+from bosdyn.api import image_pb2
+from google.protobuf.duration_pb2 import Duration
+
+from .spot_config import *
+
+
+class AsyncImageService(AsyncPeriodicQuery):
+    """Class to get images at regular intervals.  get_image_from_sources_async query sent to the robot at every tick.  Callback registered to defined callback function.
+
+    Attributes:
+        client: The Client to a service on the robot
+        logger: Logger object
+        rate: Rate (Hz) to trigger the query
+        callback: Callback function to call when the results of the query are available
+    """
+
+    def __init__(self, client, logger, rate, callback, image_requests):
+        super(AsyncImageService, self).__init__(
+            "robot_image_service", client, logger, period_sec=1.0 / max(rate, 1.0)
+        )
+        self._callback = None
+        if rate > 0.0:
+            self._callback = callback
+        self._image_requests = image_requests
+
+    def _start_query(self):
+        if self._callback:
+            callback_future = self._client.get_image_async(self._image_requests)
+            callback_future.add_done_callback(self._callback)
+            return callback_future
+
+
+class SpotArm:
+    def __init__(
+        self,
+        robot: Robot,
+        logger: logging.Logger,
+        robot_params: typing.Dict[str, typing.Any],
+        robot_clients: typing.Dict[str, typing.Any],
+    ):
+        """
+        Constructor for SpotArm class.
+        :param robot: Robot object
+        :param logger: Logger object
+        :param robot_params: Dictionary of robot parameters
+            - robot_params['is_standing']: True if robot is standing, False otherwise
+        :param robot_clients: Dictionary of robot clients
+            - robot_clients['robot_command_client']: RobotCommandClient object
+            - robot_clients['robot_command_method']: RobotCommand method
+        """
```

Comment:
```suggestion
        Constructor for SpotArm class.
        Args:
            robot: Robot object
            logger: Logger object
            robot_params: Dictionary of robot parameters
                          - robot_params['is_standing']: True if robot is standing, False otherwise
            robot_clients: Dictionary of robot clients
                           - robot_clients['robot_command_client']: RobotCommandClient object
                           - robot_clients['robot_command_method']: RobotCommand method
```

Google doc style rather than reST.

Reply:
Agreed, done in [392da89](https://github.com/bdaiinstitute/spot_wrapper/pull/6/commits/392da898d027fc716a17f1cd7ff487183cc192cb)